heapster-saw/src/Verifier/SAW/Heapster/LLVMGlobalConst.hs

```diff
@@ -86,12 +87,12 @@ traceAndZeroM msg =
 -- | Helper function to pretty-print the value of a global
 ppLLVMValue :: L.Value -> String
 ppLLVMValue val =
-  L.withConfig (L.Config True True True) (show $ PPHPJ.nest 2 $ L.ppValue val)
+  ppLLVMLatest (show $ PPHPJ.nest 2 $ L.ppValue val)
```

Comment:
This could be made even simpler as:

```hs
ppLLVMValue :: L.Value -> String
ppLLVMValue val =
  show $ PPHPJ.nest 2 $ ppValue val
```

Where `ppValue` comes from `Lang.Crucible.LLVM.PrettyPrint`.